components/NotificationSnackBar.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <v-snackbar v-model="showSnackBar">
+        {{snackbarText}}
+        <template v-slot:action="{ attrs }">
+            <v-btn color="rgb(57, 73, 171)" text v-bind="attrs" @click="() => {$emit('closeSnackbar')}">
```

Comment:
nit: Can we use the Material Colors? https://v15.vuetifyjs.com/en/framework/colors#material-colors

```suggestion
            <v-btn color="indigo darken-1" text v-bind="attrs" @click="() => {$emit('closeSnackbar')}">
```

Reply:
I was looking to keep all the colors in accordance with the current design.

Any suggestion on colors is welcome.